.github/workflows/build-wheels-platforms.yml

```diff
@@ -0,0 +1,191 @@
+name: platforms-dispatches
+
+on:
+  # TODO schedule on final merge
```

Comment:
I am not sure if we want to suspend the "old wheels build"?

Because if we do the schedule, both builds will run together. And now for the "new wheels build" (this PR) there is only Python 3.8 but much more wheels and for the "old wheels build" there are more Python versions but only some wheels.